### PR TITLE
Remove pseudo nodes that are made unconnected by filtering.

### LINKED ISSUE #483: When filtering out system containers, we should also filter out pseudo nodes if they were only connected to system containers.
Below, the internet pseudo node should be filtered out:

screen shot 2015-09-14 at 13 14 10
screen shot 2015-09-14 at 13 14 24

### PR DESCRIPTION
**Pull request by tomwilkie**
Fixes #483

### CODE REVIEW

File: render/render.go [hunk 1]
```diff
@@ -178,10 +178,12 @@ type Filter struct {
 // Render implements Renderer
 func (f Filter) Render(rpt report.Report) RenderableNodes {
 	output := RenderableNodes{}
+	inDegrees := map[string]int{}
 	for id, node := range f.Renderer.Render(rpt) {
 		if f.FilterFunc(node) {
 			output[id] = node
 		}
+		inDegrees[id] = 0
```
Comment (peterbourgon) [inline render/render.go]: Is this like the initial set? If so, it's not necessary: the zero value of an int is zero.

**Comment by peterbourgon** [comment]
LGTM.

File: render/render.go (hunk 1 above)
Comment (tomwilkie) [inline render/render.go]: Indeed, except if we didn't do the initial set the iteration through all the in degrees below would only yield those with in degree > 0, and we wouldn't filter out those with no incoming edges...

Comment (peterbourgon) [inline render/render.go]: Gotcha.

**Pull request merged**